platform/kube/client.go

```diff
@@ -303,18 +304,18 @@ func kindToAPIName(s string) string {
 }
 
 func kubeToModel(kind string, schema model.ProtoSchema, config *Config) (*model.Config, error) {
-	// TODO: validate incoming kube object
+	// TODO: validate incoming kube object	
 	spec, err := mapToProto(schema.MessageName, config.Spec)
 	if err != nil {
 		return nil, err
 	}
-	var status proto.Message
-	if config.Status != nil {
-		status, err = mapToProto(schema.StatusMessageName, config.Status)
-		if err != nil {
-			return nil, err
-		}
-	}
+	// var status proto.Message
+	// if config.Status != nil {
+	// 	status, err = mapToProto(schema.StatusMessageName, config.Status)
+	// 	if err != nil {
+	// 		return nil, err
+	// 	}
+	// }
```

Comment:
leave this code as is, you can use a dummy proto for status (like the one in mock_config). I'm pretty sure we will need this later but it doesn't hurt now.